src/node-exporter/deploy/node-exporter.yaml.template

```diff
@@ -78,6 +78,8 @@ spec:
           - '--no-collector.wifi'
           - '--no-collector.xfs'
           - '--no-collector.zfs'
+          - '--collector.filesystem.ignored-fs-types'
+          - 'autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs|tmpfs|cifs|nfs4?|lxcfs'
```

Comment:
use `'^(autofs|...|lxcfs)$'` ?

Reply:
I'm not sure, in that case lxcfs should be `fuse.lxcfs`, not sure if others are same. Since they are from default value, I think it's ok.

Reply:
I found the [default values](https://github.com/prometheus/node_exporter/blob/v0.18.0/collector/filesystem_linux.go#L33) are limited by `^(...)$`, so maybe keep the origin?

Reply:
ah, did not see that.